api/src/main/java/org/infinispan/api/async/StrongCounters.java

```diff
@@ -0,0 +1,19 @@
+package org.infinispan.api.async;
+
+import java.util.concurrent.CompletionStage;
+
+import org.infinispan.api.common.AsyncIterator;
+import org.infinispan.api.configuration.CounterConfiguration;
+
+/**
+ * @since 13.0
+ **/
+public interface StrongCounters {
+   CompletionStage<StrongCounter> get(String name);
```

Comment:
would make sense these kinds of method to return `CompletionStage<Optional<StrongCounter>>`?

Reply:
I've come to dislike `Optional` quite a bit :)

Reply:
This brings up a good point though. What do we do if the cache isn't defined? Optional, null or an exception? We removed default, so we can't just create one.